 Network vhogemann_default  Creating
 Network vhogemann_default  Created
 Container vhogemann-postgres-1  Creating
 Container vhogemann-postgres-1  Created
 Container vhogemann-api1-1  Creating
 Container vhogemann-api2-1  Creating
 Container vhogemann-api1-1  Created
 Container vhogemann-api2-1  Created
 Container vhogemann-nginx-1  Creating
 Container vhogemann-nginx-1  Created
Attaching to api1-1, api2-1, nginx-1, postgres-1
postgres-1  | The files belonging to this database system will be owned by user "postgres".
postgres-1  | This user must also own the server process.
postgres-1  | 
postgres-1  | The database cluster will be initialized with locale "en_US.utf8".
postgres-1  | The default database encoding has accordingly been set to "UTF8".
postgres-1  | The default text search configuration will be set to "english".
postgres-1  | 
postgres-1  | Data page checksums are disabled.
postgres-1  | 
postgres-1  | fixing permissions on existing directory /var/lib/postgresql/data ... ok
postgres-1  | creating subdirectories ... ok
postgres-1  | selecting dynamic shared memory implementation ... posix
postgres-1  | selecting default max_connections ... 100
postgres-1  | selecting default shared_buffers ... 128MB
postgres-1  | selecting default time zone ... Etc/UTC
postgres-1  | creating configuration files ... ok
postgres-1  | running bootstrap script ... ok
postgres-1  | performing post-bootstrap initialization ... ok
nginx-1     | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1     | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1     | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1     | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1     | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1     | /docker-entrypoint.sh: Configuration complete; ready for start up
postgres-1  | initdb: warning: enabling "trust" authentication for local connections
postgres-1  | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
postgres-1  | syncing data to disk ... ok
postgres-1  | 
postgres-1  | 
postgres-1  | Success. You can now start the database server using:
postgres-1  | 
postgres-1  |     pg_ctl -D /var/lib/postgresql/data -l logfile start
postgres-1  | 
postgres-1  | waiting for server to start....2024-02-18 19:12:12.927 UTC [49] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres-1  | 2024-02-18 19:12:12.933 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-1  | 2024-02-18 19:12:12.943 UTC [52] LOG:  database system was shut down at 2024-02-18 19:12:11 UTC
postgres-1  | 2024-02-18 19:12:12.949 UTC [49] LOG:  database system is ready to accept connections
postgres-1  |  done
postgres-1  | server started
postgres-1  | 
postgres-1  | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/bootstrap.sql
postgres-1  | CREATE TABLE
postgres-1  | CREATE INDEX
postgres-1  | CREATE TYPE
postgres-1  | CREATE TABLE
postgres-1  | CREATE INDEX
postgres-1  | CREATE INDEX
postgres-1  | CREATE PROCEDURE
postgres-1  | CREATE PROCEDURE
postgres-1  | INSERT 0 6
postgres-1  | 
postgres-1  | 
postgres-1  | waiting for server to shut down...2024-02-18 19:12:13.129 UTC [49] LOG:  received fast shutdown request
postgres-1  | .2024-02-18 19:12:13.132 UTC [49] LOG:  aborting any active transactions
postgres-1  | 2024-02-18 19:12:13.134 UTC [49] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
postgres-1  | 2024-02-18 19:12:13.134 UTC [50] LOG:  shutting down
postgres-1  | 2024-02-18 19:12:13.136 UTC [50] LOG:  checkpoint starting: shutdown immediate
postgres-1  | 2024-02-18 19:12:13.322 UTC [50] LOG:  checkpoint complete: wrote 91 buffers (0.6%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.009 s, sync=0.165 s, total=0.188 s; sync files=73, longest=0.119 s, average=0.003 s; distance=268 kB, estimate=268 kB; lsn=0/152DAC0, redo lsn=0/152DAC0
postgres-1  | 2024-02-18 19:12:13.325 UTC [49] LOG:  database system is shut down
postgres-1  |  done
postgres-1  | server stopped
postgres-1  | 
postgres-1  | PostgreSQL init process complete; ready for start up.
postgres-1  | 
postgres-1  | 2024-02-18 19:12:13.353 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
postgres-1  | 2024-02-18 19:12:13.353 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres-1  | 2024-02-18 19:12:13.353 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres-1  | 2024-02-18 19:12:13.358 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres-1  | 2024-02-18 19:12:13.366 UTC [65] LOG:  database system was shut down at 2024-02-18 19:12:13 UTC
postgres-1  | 2024-02-18 19:12:13.373 UTC [1] LOG:  database system is ready to accept connections
postgres-1  | 2024-02-18 19:12:15.873 UTC [77] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:20.963 UTC [86] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:26.046 UTC [99] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:31.125 UTC [107] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:36.197 UTC [115] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:41.277 UTC [123] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:41.941 UTC [69] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:41.941 UTC [69] DETAIL:  Failing row contains (1, 1, -104392, 100000).
postgres-1  | 2024-02-18 19:12:41.941 UTC [69] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:41.941 UTC [69] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:42.084 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:42.084 UTC [89] DETAIL:  Failing row contains (1, 1, -102613, 100000).
postgres-1  | 2024-02-18 19:12:42.084 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:42.084 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:42.228 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:42.228 UTC [88] DETAIL:  Failing row contains (1, 1, -103513, 100000).
postgres-1  | 2024-02-18 19:12:42.228 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:42.228 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:42.606 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:42.606 UTC [88] DETAIL:  Failing row contains (1, 1, -108396, 100000).
postgres-1  | 2024-02-18 19:12:42.606 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:42.606 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:42.640 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:42.640 UTC [89] DETAIL:  Failing row contains (1, 1, -100742, 100000).
postgres-1  | 2024-02-18 19:12:42.640 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:42.640 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:42.675 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:42.675 UTC [90] DETAIL:  Failing row contains (1, 1, -103020, 100000).
postgres-1  | 2024-02-18 19:12:42.675 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:42.675 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:42.709 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:42.709 UTC [78] DETAIL:  Failing row contains (2, 2, -81479, 80000).
postgres-1  | 2024-02-18 19:12:42.709 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:42.709 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:43.323 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:43.323 UTC [91] DETAIL:  Failing row contains (1, 1, -101892, 100000).
postgres-1  | 2024-02-18 19:12:43.323 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:43.323 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:43.840 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:43.840 UTC [78] DETAIL:  Failing row contains (2, 2, -80941, 80000).
postgres-1  | 2024-02-18 19:12:43.840 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:43.840 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:44.408 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:44.408 UTC [91] DETAIL:  Failing row contains (2, 2, -87400, 80000).
postgres-1  | 2024-02-18 19:12:44.408 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:44.408 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:44.742 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:44.742 UTC [90] DETAIL:  Failing row contains (1, 1, -106349, 100000).
postgres-1  | 2024-02-18 19:12:44.742 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:44.742 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:45.105 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:45.105 UTC [89] DETAIL:  Failing row contains (1, 1, -100410, 100000).
postgres-1  | 2024-02-18 19:12:45.105 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:45.105 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:46.141 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:46.141 UTC [89] DETAIL:  Failing row contains (2, 2, -81871, 80000).
postgres-1  | 2024-02-18 19:12:46.141 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:46.141 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:46.350 UTC [132] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:46.739 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:46.739 UTC [87] DETAIL:  Failing row contains (1, 1, -104083, 100000).
postgres-1  | 2024-02-18 19:12:46.739 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:46.739 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:46.901 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:46.901 UTC [78] DETAIL:  Failing row contains (1, 1, -100622, 100000).
postgres-1  | 2024-02-18 19:12:46.901 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:46.901 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.036 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.036 UTC [87] DETAIL:  Failing row contains (2, 2, -82384, 80000).
postgres-1  | 2024-02-18 19:12:47.036 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.036 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.331 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.331 UTC [87] DETAIL:  Failing row contains (1, 1, -105849, 100000).
postgres-1  | 2024-02-18 19:12:47.331 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.331 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.542 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.542 UTC [87] DETAIL:  Failing row contains (1, 1, -107549, 100000).
postgres-1  | 2024-02-18 19:12:47.542 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.542 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.674 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.674 UTC [88] DETAIL:  Failing row contains (1, 1, -104290, 100000).
postgres-1  | 2024-02-18 19:12:47.674 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.674 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.753 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.753 UTC [91] DETAIL:  Failing row contains (1, 1, -104959, 100000).
postgres-1  | 2024-02-18 19:12:47.753 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.753 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.778 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.778 UTC [88] DETAIL:  Failing row contains (1, 1, -103636, 100000).
postgres-1  | 2024-02-18 19:12:47.778 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.778 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:47.962 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:47.962 UTC [89] DETAIL:  Failing row contains (1, 1, -104011, 100000).
postgres-1  | 2024-02-18 19:12:47.962 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:47.962 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.069 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.069 UTC [88] DETAIL:  Failing row contains (1, 1, -108258, 100000).
postgres-1  | 2024-02-18 19:12:48.069 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.069 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.095 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.095 UTC [89] DETAIL:  Failing row contains (2, 2, -80618, 80000).
postgres-1  | 2024-02-18 19:12:48.095 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.095 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.276 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.276 UTC [78] DETAIL:  Failing row contains (1, 1, -100103, 100000).
postgres-1  | 2024-02-18 19:12:48.276 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.276 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.401 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.401 UTC [87] DETAIL:  Failing row contains (1, 1, -106716, 100000).
postgres-1  | 2024-02-18 19:12:48.401 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.401 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.476 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.476 UTC [78] DETAIL:  Failing row contains (1, 1, -107200, 100000).
postgres-1  | 2024-02-18 19:12:48.476 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.476 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.526 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.526 UTC [90] DETAIL:  Failing row contains (1, 1, -105252, 100000).
postgres-1  | 2024-02-18 19:12:48.526 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.526 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:48.676 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:48.676 UTC [91] DETAIL:  Failing row contains (2, 2, -82118, 80000).
postgres-1  | 2024-02-18 19:12:48.676 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:48.676 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.027 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.027 UTC [91] DETAIL:  Failing row contains (1, 1, -101865, 100000).
postgres-1  | 2024-02-18 19:12:49.027 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.027 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.052 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.052 UTC [88] DETAIL:  Failing row contains (1, 1, -104100, 100000).
postgres-1  | 2024-02-18 19:12:49.052 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.052 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.076 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.076 UTC [90] DETAIL:  Failing row contains (1, 1, -104104, 100000).
postgres-1  | 2024-02-18 19:12:49.076 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.076 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.249 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.249 UTC [90] DETAIL:  Failing row contains (1, 1, -101014, 100000).
postgres-1  | 2024-02-18 19:12:49.249 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.249 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.368 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.368 UTC [78] DETAIL:  Failing row contains (1, 1, -106548, 100000).
postgres-1  | 2024-02-18 19:12:49.368 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.368 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.654 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.654 UTC [78] DETAIL:  Failing row contains (1, 1, -101394, 100000).
postgres-1  | 2024-02-18 19:12:49.654 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.654 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.702 UTC [69] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.702 UTC [69] DETAIL:  Failing row contains (1, 1, -108113, 100000).
postgres-1  | 2024-02-18 19:12:49.702 UTC [69] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.702 UTC [69] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.727 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.727 UTC [90] DETAIL:  Failing row contains (1, 1, -101351, 100000).
postgres-1  | 2024-02-18 19:12:49.727 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.727 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.821 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.821 UTC [78] DETAIL:  Failing row contains (1, 1, -100649, 100000).
postgres-1  | 2024-02-18 19:12:49.821 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.821 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.845 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.845 UTC [88] DETAIL:  Failing row contains (1, 1, -106153, 100000).
postgres-1  | 2024-02-18 19:12:49.845 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.845 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:49.964 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:49.964 UTC [88] DETAIL:  Failing row contains (1, 1, -103673, 100000).
postgres-1  | 2024-02-18 19:12:49.964 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:49.964 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:50.248 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:50.248 UTC [78] DETAIL:  Failing row contains (1, 1, -103780, 100000).
postgres-1  | 2024-02-18 19:12:50.248 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:50.248 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:50.521 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:50.521 UTC [87] DETAIL:  Failing row contains (1, 1, -103456, 100000).
postgres-1  | 2024-02-18 19:12:50.521 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:50.521 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:50.615 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:50.615 UTC [90] DETAIL:  Failing row contains (1, 1, -106513, 100000).
postgres-1  | 2024-02-18 19:12:50.615 UTC [90] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:50.615 UTC [90] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:50.715 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:50.715 UTC [78] DETAIL:  Failing row contains (1, 1, -100276, 100000).
postgres-1  | 2024-02-18 19:12:50.715 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:50.715 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:50.815 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:50.815 UTC [91] DETAIL:  Failing row contains (1, 1, -101314, 100000).
postgres-1  | 2024-02-18 19:12:50.815 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:50.815 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:50.999 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:50.999 UTC [78] DETAIL:  Failing row contains (1, 1, -104549, 100000).
postgres-1  | 2024-02-18 19:12:50.999 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:50.999 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:51.067 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:51.067 UTC [87] DETAIL:  Failing row contains (1, 1, -105042, 100000).
postgres-1  | 2024-02-18 19:12:51.067 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:51.067 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:51.112 UTC [69] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:51.112 UTC [69] DETAIL:  Failing row contains (1, 1, -104698, 100000).
postgres-1  | 2024-02-18 19:12:51.112 UTC [69] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:51.112 UTC [69] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:51.423 UTC [140] FATAL:  role "root" does not exist
postgres-1  | 2024-02-18 19:12:51.552 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:51.552 UTC [87] DETAIL:  Failing row contains (2, 2, -81446, 80000).
postgres-1  | 2024-02-18 19:12:51.552 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:51.552 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.073 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.073 UTC [78] DETAIL:  Failing row contains (1, 1, -101785, 100000).
postgres-1  | 2024-02-18 19:12:52.073 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.073 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.160 UTC [78] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.160 UTC [78] DETAIL:  Failing row contains (1, 1, -105186, 100000).
postgres-1  | 2024-02-18 19:12:52.160 UTC [78] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.160 UTC [78] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.417 UTC [91] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.417 UTC [91] DETAIL:  Failing row contains (2, 2, -84073, 80000).
postgres-1  | 2024-02-18 19:12:52.417 UTC [91] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.417 UTC [91] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.515 UTC [69] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.515 UTC [69] DETAIL:  Failing row contains (1, 1, -104048, 100000).
postgres-1  | 2024-02-18 19:12:52.515 UTC [69] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.515 UTC [69] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.715 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.715 UTC [88] DETAIL:  Failing row contains (1, 1, -100743, 100000).
postgres-1  | 2024-02-18 19:12:52.715 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.715 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.736 UTC [89] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.736 UTC [89] DETAIL:  Failing row contains (2, 2, -80701, 80000).
postgres-1  | 2024-02-18 19:12:52.736 UTC [89] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.736 UTC [89] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:52.816 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:52.816 UTC [87] DETAIL:  Failing row contains (1, 1, -103959, 100000).
postgres-1  | 2024-02-18 19:12:52.816 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:52.816 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:53.119 UTC [87] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:53.119 UTC [87] DETAIL:  Failing row contains (2, 2, -85194, 80000).
postgres-1  | 2024-02-18 19:12:53.119 UTC [87] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:53.119 UTC [87] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:53.162 UTC [88] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:53.162 UTC [88] DETAIL:  Failing row contains (2, 2, -87526, 80000).
postgres-1  | 2024-02-18 19:12:53.162 UTC [88] CONTEXT:  SQL statement "UPDATE balance
postgres-1  | 	    SET amount = amount - w_amount
postgres-1  | 	    WHERE client_id = a_client_id"
postgres-1  | 	PL/pgSQL function withdrawal(integer,integer,text) line 3 at SQL statement
postgres-1  | 2024-02-18 19:12:53.162 UTC [88] STATEMENT:  CALL withdrawal($1, $2, $3)
postgres-1  | 2024-02-18 19:12:53.183 UTC [90] ERROR:  new row for relation "balance" violates check constraint "amount_overdraft_limit_check"
postgres-1  | 2024-02-18 19:12:53.183 UTC [90] DETAIL:  Failing row contains (2, 2, -86096, 80000).
